jni_gen/test/test_data/test_config.yaml

```diff
@@ -0,0 +1,19 @@
+summarizer:
+  use_asm: false
```

Comment:
Are we expecting more than two frontends?
If so, make this `frontend` or `parser` with two options `doclet` and `asm`. (And probably the same change with the command line options.)

Reply:
Ah I did the change but forgot to change in this yaml.

Btw it is just a test config yaml which is not used for running, only in config provider's unit test. Sorry for the confusion I have to rename it or just use jackson_core_test/jnigen.yaml.